server/profile.js

```diff
@@ -6,6 +6,12 @@ const pool = new Pool({
     database: process.env.database,
     password: process.env.password,
     port: process.env.port,
+    // user: 'master',
+    // host: 'aoc-db-dev.cjynw6x3q1ly.us-west-1.rds.amazonaws.com',
+    // database: 'aocdb',
```

Comment:
lmao at least I wasn't the one to leak things this time!!!!! We're gonna have to rotate the db creds lol

Reply:
😥😥